source/manual/get-started.html.md

```diff
@@ -17,6 +17,12 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
 [GDS]: https://gds.blog.gov.uk/about/
 [overview-slides]: https://docs.google.com/presentation/d/1nAE65Og04JYNAc0VjYaUYLqNLuUOM9r3Mvo0PGFy_Zk
 
+## 0. Install the [Homebrew package manager](https://brew.sh) (on macOS or Linux)
```

Comment:
Zero-based indexing... I like it 😂 though should probably make this `1.` and update all the other numbers.

Reply:
Given I've basically updated every other number except the current `1.`, I suppose so.